packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -1139,6 +1139,43 @@ new ecs.Ec2Service(this, 'EC2Service', {
 });
 ```
 
+### Cluster Default Provider Strategy
+
+A capacity provider strategy is part of the configuration of a cluster, service, or task. A capacity provider strategy consists of one or more capacity providers. You can specify an optional base and weight value for finer control.
```

Comment:
```suggestion
A capacity provider strategy determines whether ECS tasks are launched on EC2 instances or Fargate/Fargate Spot. It can be specified at the cluster, service, or task level, and consists of one or more capacity providers. You can specify an optional base and weight value for finer control of how tasks are launched. The `base` specifies a minimum number of tasks on one capacity provider, and the `weight`s of each capacity provider determine how tasks are distributed after `base` is satisfied.
```